pyiron_atomistics/sphinx/output_parser.py

```diff
@@ -386,3 +389,103 @@ def get_convergence(self):
     def get_fermi(self):
         pattern = r"Fermi energy:\s+(\d+\.\d+)\s+eV"
         return np.array(re.findall(pattern, self.log_main)).astype(float)
+
+
+class SphinxWavesParser:
+    """ Class to read SPHInX waves.sxb files (HDF5 format)
+    
+        Initialize with waves.sxb filename, or use load ()
+    
+    """
+    
+    def __init__(self, file_name="waves.sxb", cwd="."):
+        """
+        Args:
+            file_name (str): file name
+            cwd (str): directory path
+        """
+        path = Path(cwd) / Path(file_name)
+        self.load(path)
+        
+
+    def load(self, filename):
+        """ Load a waves.sxb file (HDF5 format)
+        
+            filename: file name
+        """
+        self.wfile = h5py.File (filename)
+        self._eps = None
+        self._read ()
+
+    def _read(self):
+        self._check_loaded ()
```

Comment:
This does not exist anymore right?

Reply:
Actually I see that there’s still `_check_loaded` everywhere

Reply:
yes. removed it.